debezium-connector-binlog/src/main/java/io/debezium/connector/binlog/jdbc/BinlogValueConverters.java

```diff
@@ -105,15 +107,18 @@ public abstract class BinlogValueConverters extends JdbcValueConverters {
      * @param binaryHandlingMode how binary columns should be treated
      * @param adjuster a temporal adjuster to make a database specific time before conversion
      * @param eventConvertingFailureHandlingMode how to handle conversion failures
+     * @param charsetRegistry the character set registry
      */
     public BinlogValueConverters(DecimalMode decimalMode,
                                  TemporalPrecisionMode temporalPrecisionMode,
                                  BigIntUnsignedMode bigIntUnsignedMode,
                                  BinaryHandlingMode binaryHandlingMode,
                                  TemporalAdjuster adjuster,
-                                 EventConvertingFailureHandlingMode eventConvertingFailureHandlingMode) {
+                                 EventConvertingFailureHandlingMode eventConvertingFailureHandlingMode,
+                                 BinlogCharsetRegistry charsetRegistry) {
```

Comment:
Here I'd prefer passing of the registry itself so in future we can just read from the reagistry instead of changing the signature.